geth/node/node.go

```diff
@@ -179,6 +179,13 @@ func activateEthService(stack *node.Node, config *params.NodeConfig) error {
 		lightEth, err := les.New(ctx, &ethConf)
 		if err == nil {
 			updateCHT(lightEth, config)
+
```

Comment:
Can we add a comment that it's a temporary solution and should be fixed eventually, please?

Reply:
Comment added.